system-x/services/horreum/pom.xml

```diff
@@ -0,0 +1,74 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<parent>
+		<groupId>software.tnb</groupId>
+		<artifactId>system-x-services</artifactId>
+		<version>1.0-SNAPSHOT</version>
+	</parent>
+	<modelVersion>4.0.0</modelVersion>
+	<artifactId>system-x-horreum</artifactId>
+	<version>1.0-SNAPSHOT</version>
+	<name>TNB :: System-X :: Services :: Horreum</name>
+
+	<properties>
+		<gson-fire-version>1.8.5</gson-fire-version>
+		<swagger-core-version>1.6.3</swagger-core-version>
+		<okhttp-version>4.9.2</okhttp-version>
+		<gson-version>2.8.8</gson-version>
+		<commons-lang3-version>3.12.0</commons-lang3-version>
+		<jackson-databind-nullable-version>0.2.2</jackson-databind-nullable-version>
+		<threetenbp-version>1.5.0</threetenbp-version>
+		<jakarta-annotation-version>1.3.5</jakarta-annotation-version>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>io.swagger</groupId>
+			<artifactId>swagger-annotations</artifactId>
+			<version>${swagger-core-version}</version>
+		</dependency>
+		<dependency>
+			<groupId>com.google.code.findbugs</groupId>
+			<artifactId>jsr305</artifactId>
+			<version>3.0.2</version>
+		</dependency>
+		<dependency>
+			<groupId>com.squareup.okhttp3</groupId>
+			<artifactId>okhttp</artifactId>
+			<version>${okhttp-version}</version>
+		</dependency>
+		<dependency>
+			<groupId>com.squareup.okhttp3</groupId>
+			<artifactId>logging-interceptor</artifactId>
+			<version>${okhttp-version}</version>
+		</dependency>
+		<dependency>
+			<groupId>com.google.code.gson</groupId>
+			<artifactId>gson</artifactId>
+			<version>${gson-version}</version>
+		</dependency>
+		<dependency>
+			<groupId>io.gsonfire</groupId>
+			<artifactId>gson-fire</artifactId>
+			<version>${gson-fire-version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-lang3</artifactId>
+			<version>${commons-lang3-version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.threeten</groupId>
+			<artifactId>threetenbp</artifactId>
+			<version>${threetenbp-version}</version>
+		</dependency>
+		<dependency>
+			<groupId>jakarta.annotation</groupId>
+			<artifactId>jakarta.annotation-api</artifactId>
+			<version>${jakarta-annotation-version}</version>
+			<scope>provided</scope>
+		</dependency>
```

Comment:
needed?

Reply:
the generated code doesn't compile if you remove that